compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEBufferizeToAllocation.cpp

```diff
@@ -120,10 +120,14 @@ void AMDAIEBufferizeToAllocationPass::runOnOperation() {
 
 }  // namespace
 
+std::unique_ptr<Pass> createAMDAIEBufferizeToAllocationPass() {
```

Comment:
I think you can combine these two to
```
std::unique_ptr<Pass> create*Pass(AMDAIE*Options options = {});
```

The default initializer should set the fields to the default values.

Reply:
Really nice! Thanks.